### PR TITLE
feat: expose text format errors

### PR DESCRIPTION
**Pull request by aslafy-z**
Give users a way to get humanized errors as string.

Leverage [`ajv's errorsText() method`](https://github.com/epoberezkin/ajv#errorstextarrayobject-errors--object-options---string)

- expose `errorsText` function
- use `errorsText` in the cli output

### CODE REVIEW

File: src/lib/missing-transition-target.js [hunk 1]
```diff
@@ -15,17 +15,11 @@ module.exports = (definition) => {
 
   // check if all states are reachable
   const unreachable = machineStates.filter(state => reachableStates.indexOf(state) === -1)
-    .map(state => ({
-      'Error code': 'MISSING_TRANSITION_TARGET',
-      Message: `State ${state} is not reachable`,
-    }));
+    .map(state => `State ${state} is not reachable`);
```
Comment (ChristopheBougere) [inline src/lib/missing-transition-target.js]: This looks like a unnecessary breaking change, couldn't you keep it as before and do `missingTransitionTargetErrors.map(e => e.Message).join(separator)` in the validator instead ?
Or even ```missingTransitionTargetErrors.map(e => `${e.Code}: ${e.Message}`).join(separator)```

**Comment by aslafy-z** [comment]
Thank you for the review. I applied some of the proposed fixes and will complete this PR in few days.

**Comment by aslafy-z** [comment]
I'm not convinced of this api but I'm not sure how to do it another way since ajv has a particular output that does not match yours.. Let me know if you have suggestions.

**Comment by ChristopheBougere** [comment]
Just released in 1.7.0
Thanks for contributing, the CLI output is much more readable now 👌

**Pull request merged**